### PR TITLE
 windows cmdutil

### PR DESCRIPTION
**Pull request by alexadhy**
Related to:

- https://github.com/skycoin/skywire/issues/778	
- https://github.com/skycoin/skywire/pull/779

 Changes:	
- per platform `syslog` and syscall signals definition in `cmdutil`.

How to test this PR:

- Checkout this PR
- Checkout https://github.com/skycoin/skywire/pull/779
- `go mod edit -replace github.com/skycoin/dmsg=../dmsg`
- build the `skywire` binaries via `make`
- test it on windows

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ linters-settings:
     #  2. you use go >= 1.10
     #  3. you do repeated runs (false for CI) or cache $GOPATH/pkg or `go env GOCACHE` dir in CI.
     use-installed-packages: false
-  golint:
+  revive:
```
Comment (jdknives) [inline .golangci.yml:74]: Any reason we are switching? Is revive running a superset of linting rules?

**Review by jdknives — APPROVED** [review]
Cant test it but it looks good to me from first glance. Someone needs to verify it builds and works on Windows.

File: .golangci.yml (hunk 1 above)
Comment (alexadhy) [reply .golangci.yml:74]: golint is deprecated (repository is archived, and latest version of golangci-lint uses revive to replace it). revive has the same feature as golint, not going to break anything I think

File: cmdutil/signal_context.go [hunk 2]
```diff
@@ -18,7 +17,8 @@ func SignalContext(ctx context.Context, log logrus.FieldLogger) (context.Context
 	ctx, cancel := context.WithCancel(ctx)
 
 	ch := make(chan os.Signal)
-	signal.Notify(ch, []os.Signal{syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT}...)
+	ignoredSigs := ignoreSignals()
```
Comment (i-hate-nicknames) [inline cmdutil/signal_context.go]: I believe those are not ignored signals, but vice versa - the signals we want to listen to

**Review by i-hate-nicknames — COMMENTED** [review]
Good job, got just a small comment. (I haven't run it on windows, just read the changes)

File: cmdutil/signal_context.go (hunk 2 above)
Comment (alexadhy) [reply cmdutil/signal_context.go]: yeah those are the signal we are listening to, I think I may have typo it, sorry

**Comment by alexadhy** [comment]
can you merge this?

**Review by jdknives — APPROVED** [review]

**Pull request merged**